components/automate-ui/src/app/pages/automate-settings/automate-settings.component.scss

```diff
@@ -1,35 +1,104 @@
 @import "~styles/variables";
 
-chef-td {
-  &.description {
-    flex-basis: 45%;
+.container {
+  background-color: $chef-white;
+}
+
+h2 {
+  font-size: 16px;
+  font-weight: 600;
+}
+
+.group-section {
+  margin-bottom: 30px;
+
+  chef-button {
+    margin: 0;
   }
 
-  &.input {
-    flex: none;
+  .checkbox-row {
+    display: flex;
+
+    input {
+      width: 60px;
+    }
+
+    p {
+      margin: 0;
+    }
+
+    &+.checkbox-row {
+      margin-top: 12px;
+
+      &.secondary-row {
+        margin-left: 69px;
+      }
+    }
+
+
+    .checkbox-container {
+      margin-top: 12px;
+    }
+
+    chef-checkbox {
+      margin-right: 8px;
+      font-size: 17px;
+    }
   }
 }
 
-input {
-  width: 60px;
-  height: 100%;
+.chef-input {
+  margin: 0 8px;
 }
 
-chef-select {
-  width: 120px;
+chef-checkbox {
+  ::ng-deep .mat-checkbox-layout {
+    padding-bottom: 0;
+  }
 }
 
-.center {
-  margin: auto;
-  padding: 10px;
-  width: 10%;
-}
 
-chef-checkbox {
-  font-size: 17px;
-  width: 110px;
 
-  ::ng-deep .mat-checkbox-layout {
+// temporary - needs to go in theme file for material
+mat-form-field {
+  margin: 0 8px 0 0;
+  width: 120px;
+
+  ::ng-deep .mat-form-field-wrapper {
     padding-bottom: 0;
   }
+
+  ::ng-deep .mat-form-field-infix {
+    padding: 0;
+    border-top: 0;
+  }
+
+  ::ng-deep .mat-select-disabled {
+    background-color: $chef-light-grey;
+    opacity: 0.5;
+
+    .mat-select-value {
+      color: rgb(84, 84, 84);
+    }
+  }
+}
+
+[panelClass="chef-dropdown"] {
+  height: 45px;
+  background-color: $chef-white;
+  padding: 1em;
+  font-size: 14px;
+  border: 1px solid $chef-light-grey;
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-family: inherit;
+  transition: border 0.4s ease;
+
+  &:focus,
+  &:active {
+    border-color: $chef-primary-bright;
+    outline: none;
+  }
 }
+
+// end temporary
```

Comment:
What's this communicating?

Reply:
Good catch - that was a note for me because I needed to move some Sass over to an application wide level